pom.xml

```diff
@@ -111,8 +111,9 @@
         <!-- Others -->
         <commons-lang.version>2.6</commons-lang.version>
         <commons-lang3.version>3.9</commons-lang3.version>
-        <apacheds.version>2.0.0-M21</apacheds.version>
-        <apacheds.codec.version>1.0.0-M33</apacheds.codec.version>
+        <commons-io.version>2.6</commons-io.version>
+        <apacheds.version>2.0.0-M24</apacheds.version>
+        <apacheds.codec.version>1.0.0</apacheds.codec.version>
```

Comment:
I didn't find this GAV in Wildfly.

Reply:
mvn dependency:list on wildfly didn't list a version for the codec. I selected the version for codec from the parent pom for apacheds 2.0.0-M24.